models/ResourceService.cfc

```diff
@@ -1,439 +1,517 @@
-		<cfargument name="rbFile"   required="true" 	type="string" 	hint="This must be the path + filename UP to but NOT including the locale. We auto-add .properties to the end alongside the locale">
-		<cfargument name="rbLocale" required="false"  	type="string" 	default="en_US" 	hint="The locale of the bundle to load">
-		<cfargument name="force" 	required="false"  	type="boolean" 	default="false" 	hint="Forces the loading of the bundle even if its in memory">
-		<cfargument name="rbAlias" 	required="false"  	type="string" 	default="default" 	hint="The unique alias name used to store this resource bundle in memory. The default name is the name of the rbFile passed if not passed.">
-
-		<!--- Setup rbAlias if not passed --->
-		<cfif NOT structKeyExists( arguments, "rbAlias" ) or NOT len( arguments.rbAlias )>
-			<cfset arguments.rbFile  = replace( arguments.rbFile, "\", "/", "all" )>
-			<cfset arguments.rbAlias = listLast( arguments.rbFile, "/" )>
-		</cfif>
-
-		<!--- Verify bundle register name exists --->
-		<cfif NOT structKeyExists( instance.aBundles, arguments.rbAlias )>
-			<cflock name="rbregister.#hash( arguments.rbFile & arguments.rbAlias )#" type="exclusive" timeout="10" throwontimeout="true">
-				<cfif NOT structKeyExists( instance.aBundles, arguments.rbAlias )>
-					<cfset instance.aBundles[ arguments.rbAlias ] = structnew()>
-				</cfif>
-			</cflock>
-		</cfif>
-
-		<!--- Verify bundle register locale exists or forced --->
-		<cfif NOT structKeyExists( instance.aBundles[ arguments.rbAlias ], arguments.rbLocale ) OR arguments.force>
-			<cflock name="rbload.#hash( arguments.rbFile & arguments.rbLocale )#" type="exclusive" timeout="10" throwontimeout="true">
-				<cfscript>
-				if( NOT structKeyExists( instance.aBundles[ arguments.rbAlias ], arguments.rbLocale ) OR arguments.force ){
+		}
+
+		// Verify bundle register locale exists or forced
+		if ( !structKeyExists( variables.aBundles[ arguments.rbAlias ], arguments.rbLocale ) || arguments.force ) {
+			lock
+				name          ="rbload.#hash( arguments.rbFile & arguments.rbLocale )#"
+				type          ="exclusive"
+				timeout       ="10"
+				throwontimeout="true" {
+				if ( !structKeyExists( variables.aBundles[ arguments.rbAlias ], arguments.rbLocale ) || arguments.force ) {
 					// load a bundle and store it.
-					instance.aBundles[ arguments.rbAlias ][ arguments.rbLocale ] = getResourceBundle(rbFile=arguments.rbFile, rbLocale=arguments.rbLocale);
+					variables.aBundles[ arguments.rbAlias ][ arguments.rbLocale ] = getResourceBundle(
+						rbFile   = arguments.rbFile,
+						rbLocale = arguments.rbLocale
+					);
 					// logging
-					if( log.canDebug() ){
-						log.debug( "Loaded bundle: #arguments.rbFile#:#arguments.rbAlias# for locale: #arguments.rbLocale#, forced: #arguments.force#" );
+					if ( log.canDebug() ) {
```

Comment:
I think this is important to denote in your logs, Can you change this to `info` level please

Reply:
done